numba/targets/cpu.py

```diff
@@ -126,7 +126,11 @@ def build_list(self, builder, list_type, items):
         """
         Build a list from the Numba *list_type* and its initial *items*.
         """
-        return listobj.build_list(self, builder, list_type, items)
+        if self._disable_reflected_list:
+            from numba import listobject
+            return listobject.build_list(self, builder, list_type, items)
+        else:
+            return listobj.build_list(self, builder, list_type, items)
```

Comment:
I think to force thinking about this there should be no module level list impl import. Perhaps do:
```python
        if self._disable_reflected_list:
            from numba import listobject as listobj_impl
        else:
            from numba.targets import listobj as listobj_impl
        return listobj_impl.build_list(self, builder, list_type, items)
```
and remove the top level import of `listobj`?

Reply:
:+1: